x/slashing/keeper/infractions.go

```diff
@@ -58,19 +58,30 @@ func (k Keeper) HandleValidatorSignatureWithParams(ctx context.Context, params t
 	signedBlocksWindow := params.SignedBlocksWindow
 
 	// Compute the relative index, so we count the blocks the validator *should*
-	// have signed. We will use the 0-value default signing info if not present,
-	// except for start height. The index is in the range [0, SignedBlocksWindow)
+	// have signed. We will also use the 0-value default signing info if not present.
+	// The index is in the range [0, SignedBlocksWindow)
```

Comment:
Note I deleted:
`except for start height.` because I think this part of the comment is inaccurate. There is no logic anywhere here that says something to the effect `if startHeight = 0, startHeight = height`, nor any tests or logic that assume this.

Somewhat of a driveby change, but figured I should edit it as I'm making another thing depend on StartHeight.